README.md

```diff
@@ -6,7 +6,7 @@ MicroPython applications.
 
 The libraries here fall into four categories corresponding to the four top-level directories:
 
- * **python-stdlib**: Compatible versions of modules from the [Python Standard Library](https://docs.python.org/3/library/). These should be drop-in replacements for the Python libraries, although many have reduced functionality or missing methods or classes (which may not be an issue for many most cases).
+ * **python-stdlib**: Compatible versions of modules from the [Python Standard Library](https://docs.python.org/3/library/). These should be drop-in replacements for the Python libraries, although many have reduced functionality or missing methods or classes (which may not be an issue for many most cases). 
 
```

Comment:
This just adds a space?

Reply:
That wasn't intended to be there, not sure how that happenned. I used the github web editor... didn't think I went near that paragraph. I'll have to clean it up

Reply:
Ay yeah that's possible, I recall the editor doing things which it ideally shouldn't